opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Group.java

```diff
@@ -287,7 +290,7 @@ public static Group getByName(String name) {
         Group ret = null;
         RuntimeEnvironment env = RuntimeEnvironment.getInstance();
         if (env.hasGroups()) {
-            for (Group grp : env.getGroups()) {
+            for (Group grp : env.getGroups().values()) {
```

Comment:
The entire goal of the issue and therefore of this change was to avoid this loop